### DISCUSSION TITLE
need help : Code extract web scrapping website exchange rate bagian kurs (Capstone DA Web Scrapping)

### DISCUSSION THREAD

**Discussion by mgandhiardi** [teamalgoritma/community#654] [Q&A] [answered]
saya mengalami kesulitan untuk extract data web scrapping website https://www.exchange-rates.org/history/IDR/USD/T.
data yg ingin diambil adalah data nilai Rupiah nya saja. berikut gambarannya bentuk data hasil find_all()
![image](https://github.com/teamalgoritma/community/assets/130325771/5cd76e59-f1e1-48cc-8fca-368acb06e8ea)
data yg keluar berbentuk stack/berurutan dari atas kebawah,. bagaimana code looping untuk pengambilan/scrapping data Rupiahnya saja? data yang ingin diambil yaitu index = 1 , 5, 9, 13 ...... (i+4)

**Comment by nadyadtm** [comment, marked as answer]
Dear Pak @mgandhiardi ,

Untuk membuat code looping tersebut, bapak bisa menggunakan perulangan dengan cara berikut ini
```
for x in range(start, stop, step):
     .....
```

dengan keterangan
start => index awal
stop => index akhir
step => penambahan berapa (misal ingin penambahannya i + 4 maka ditulis saja 4)

Jika ingin melihat contoh penggunaannya dapat merujuk pada link berikut
https://sparkbyexamples.com/python/python-for-loop-increment/

Terima kasih atas pertanyaannya. Semoga membantu

**Reply by mgandhiardi** [reply]
terima kasih,. sudah bisa
